docs/source/conf.py

```diff
@@ -33,6 +33,7 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
+    'tools.sphinxext.c7n_schema',
```

Comment:
yeah, it should be in tools but it should be its own package there ie the python import path for tools/c7n_salactus is c7n_salactus, the top level tools directory is just that a folder.